script/scanRear.sh

```diff
@@ -12,36 +12,44 @@ if [ "$USE_JPEG_COMPRESSION" = "true" ]; then
   gm_opts+=(-compress JPEG -quality 80)
 fi
 
-# device=$1
+device="$1"
+script_dir="/opt/brother/scanner/brscan-skey/script"
+
 mkdir -p /tmp
 cd /tmp || exit
 date=$(ls -rd */ | grep "$(date +"%Y-%m-%d")" | head -1)
 date=${date%/}
-filename_base="/tmp/$date/${date}-back-page"
-output_file="${filename_base}%04d.pnm"
+tmp_dir="/tmp/${date}"
+filename_base="${tmp_dir}/${date}-back-page"
+tmp_output_file="${filename_base}%04d.pnm"
+tmp_output_pdf_file="${tmp_dir}/${date}.pdf"
+output_pdf_file="/scans/${date}.pdf"
 
-cd "/tmp/${date}" || exit
+cd "$tmp_dir" || exit
 
 pkill -P "$(cat scan_pid)"
 rm scan_pid
 
-#sthg is wrong with device name, probably escaping, use default printer:
-#scan_cmd="scanimage -l 0 -t 0 -x 215 -y 297 --device-name=$device --resolution=$resolution --batch=$output_file"
-scan_cmd="scanimage -l 0 -t 0 -x 215 -y 297 --resolution=$resolution --batch=$output_file"
+function scan_cmd() {
+  # `brother4:net1;dev0` device name gets passed to scanimage, which it refuses as an invalid device name for some reason.
+  # Let's use the default scanner for now
+  # scanimage -l 0 -t 0 -x 215 -y 297 --device-name="$1" --resolution="$2" --batch="$3"
+  scanimage -l 0 -t 0 -x 215 -y 297 --format=pnm --resolution="$2" --batch="$3"
```

Comment:
**note**:  Explicitly specify `--format=pnm` to avoid unnecessary warning fallback message.